src/ui/DynamicHierarchicalFacetSearch/DynamicHierarchicalFacetSearchValueRenderer.ts

```diff
@@ -35,7 +35,7 @@ export class DynamicHierarchicalFacetSearchValueRenderer {
   private get pathToRender(): { start: string[]; end?: string[] } {
     const parentPath = this.facetValue.fullPath.slice(0, -1);
     if (!parentPath.length) {
-      return { start: [this.facet.options.clearLabel] };
```

Comment:
What if instead of reusing the clearLabel option, we just reused the string `AllCategories`. No new strings would need to be added. It would be a 1 liner change

Reply:
Good point, I didn't think of this. I'll revert my other changes and use this solution instead.